src/Constants_Conversion.f90

```diff
@@ -27,6 +27,10 @@ MODULE Constants_Conversion
 PUBLIC :: C_to_F
 PUBLIC :: F_to_K
 PUBLIC :: K_to_F
+PUBLIC :: tempTo_K
+PUBLIC :: tempTo_C
+PUBLIC :: tempTo_F
+PUBLIC :: tempTo_R
```

Comment:
Do we use Rankine for anything?  I was just expecting K, C, and F

Reply:
I don't believe so and I also don't believe we allow for it in inputs. All the dictionary checks in VERAIn are for F, C, and K.

Reply:
as long as it's correct, don't remove it

Reply:
Yeah, it's fine.  Just didn't expect that one.